src/Sylius/Behat/Client/ApiPlatformClient.php

```diff
@@ -35,6 +35,11 @@ public function index(string $resource): void
         $this->client->request('GET', '/new-api/'.$resource, [], [], ['HTTP_ACCEPT' => 'application/ld+json']);
     }
 
+    public function show(string $resource, string $id): void
+    {
+        $this->client->request('GET', sprintf('/new-api/%s/%s', $resource, $id), [], [], ['HTTP_ACCEPT' => 'application/ld+json']);
```

Comment:
Maybe we can use `ApiPlatform\Core\Api\IriConverterInterface` to generate an iri in this client?

Reply:
I agree :+1: However, do you think it can be a part of next PR, that would refactor `ApiPlatformClient` a little bit? It's needed, nevertheless